crm_phonecall/wizard/crm_phonecall_to_phonecall.py

```diff
@@ -56,16 +56,18 @@ def action_schedule(self):
         phonecall_obj = self.env['crm.phonecall']
         phonecalls = phonecall_obj.browse(self._context.get('active_ids', []))
         for this in self:
-            phonecalls.schedule_another_phonecall(
+            new_phonecalls_dict = phonecalls.schedule_another_phonecall(
                 this.date,
                 this.name,
                 this.user_id.id,
                 this.team_id.id or False,
                 this.tag_ids.ids,
                 action=this.action,
             )
-
-        return phonecalls.redirect_phonecall_view()
+        new_phonecalls = self.env['crm.phonecall']
+        for new_phonecall in new_phonecalls_dict.values():
```

Comment:
You can do `new_phonecall = reduce(lambda x, y: a + b, new_phonecalls_dict.values())` for reducing code

Reply:
done